src/operation/credential/index.ts

```diff
@@ -13,13 +13,13 @@ import { Big, Bool, IP, ShortDate, TimeStamp } from "../../types"
 export class Credential extends ContractGenerator {
     constructor(
         networkID: string,
-        contract?: string | Address,
         api?: string | IP,
     ) {
-        super(networkID, contract, api)
+        super(networkID, api)
     }
 
     createService(
+        contractAdd: string | Address,
```

Comment:
@soonkuk In `src/api/models/index.ts`, dao, nft, credential, sto, kyc, etc. are exported as objects called `contract` and use it as `contract` in each `index.ts` file. Accordingly, to avoid errors due to overlapping variable names, it was inevitably specified the name of newly added variable as `contractAdd`.

Reply:
@HayoungOh5 Ok, I understand it.
What does "Add" mean? 
I wish the variable name that replaces "contract" would represent the meaning better.

Reply:
@soonkuk  change the variable "contractAdd" to "contractAddr". If there is any issue, please merge it. Then I'll request another PR about toBuffer().

Reply:
@HayoungOh5 Ok.